### DISCUSSION TITLE
Adjusting Column Width

### DISCUSSION THREAD

**Discussion by PanduhP** [kanboard/kanboard#5430] [Q&A] [answered]
Hello!  I use Kanboard to track manufacturing processes and unfortunately I have a considerable amount of steps.  Each task has a really small amount of data, usually just the title and a few tags.

If possible, I'd like to not have to scroll horizontally to see all the steps.  Is there a way to thin the width of the columns so I can fit more columns on my screen without having to scroll?  Eventually I will get an ultrawide or super ultrawide to be able to see them all at once but I'm hoping to find a less expensive option in the interim.  Thanks!

**Comment by fguillot** [comment, marked as answer]
You can try displaying the board in compact mode. Additionally, you could attempt using a custom stylesheet to adjust the column width.

**Comment by PanduhP** [comment]
Ah ha!  Compact view is perfect, thank you!
